aleth-bootnode/main.cpp

```diff
@@ -1,6 +1,21 @@
-// Aleth: Ethereum C++ client, tools and libraries.
```

Comment:
No. These 3 lines you changed is exactly what was suppose to be used on all other places.